docs/csharp/getting-started/introduction-to-the-csharp-language-and-the-net-framework.md

```diff
@@ -63,4 +63,4 @@ C# is an elegant and type-safe object-oriented language that enables developers
 ## See also
 
 - [C#](../index.md)
-- [Getting Started with Visual C# and Visual Basic](/visualstudio/ide/getting-started-with-visual-csharp-and-visual-basic)
+- [Getting Started with Visual C# and Visual Basic](/visualstudio/ide/quickstart-visual-basic-console)
```

Comment:
Since the edited file is in csharp folder, I suggest the article to be the csharp one. I also edited the link text.

```suggestion
- [Getting Started with Visual C#](/visualstudio/ide/quickstart-csharp-console)
```